modular_zubbers/code/game/machinery/doors/door.dm

```diff
@@ -0,0 +1,42 @@
+#define LINK_DENY "<a href='?_src_=usr;open_door=[REF(src)];user=[REF(user)];action=deny'>\a (deny)</a>"
+#define LINK_OPEN "<a href='?_src_=usr;open_door=[REF(src)];user=[REF(user)];action=open'>\a (open)</a>"
+#define LINK_BOLT "<a href='?_src_=usr;open_door=[REF(src)];user=[REF(user)];action=bolt'>\a (bolt)</a>"
+#define LINK_SHOCK "<a href='?_src_=usr;open_door=[REF(src)];user=[REF(user)];action=shock'>\a (shock)</a>"
+
+/obj/machinery/door/airlock
+	//so you don't spam the AI
+	COOLDOWN_DECLARE(request_cd)
+	//so the AI doesn't spam you
+	COOLDOWN_DECLARE(answer_cd)
+
+/obj/machinery/door/airlock/attack_hand_secondary(mob/user, list/modifiers)
+	if(!COOLDOWN_FINISHED(src, request_cd))
```

Comment:
Can you have the cooldown attatch to the mob sending the request. Since you can still spam the AI with door requests on seperate doors.

Reply:
fixed